pinot-controller/src/main/resources/.prettierignore

```diff
@@ -0,0 +1 @@
+*
```

Comment:
what's the reasoning here? If anything I was hoping we'd start enforcing prettier

Reply:
I will write prettier and eslint configs properly for this project.
That will be in a separate PR, until then this should be ok.